keps/sig-cloud-provider/20190125-removing-in-tree-providers.md

```diff
@@ -180,14 +155,11 @@ With the additions needed in the short term to make this work; the Staging area
 - Apimachinery
 - Apiserver
 - Client-go
-- **cloud-provider-aws**
-- **cloud-provider-azure**
-- **cloud-provider-cloudstack**
-- **cloud-provider-gce**
-- **cloud-provider-openstack**
-- **cloud-provider-ovirt**
-- **cloud-provider-photon**
-- **cloud-provider-vsphere**
+- **legacy-cloud-provider-aws**
```

Comment:
Alternatively we can just have 1 repo `k8s.io/legacy-cloud-providers` and move each provider into their own package there.

Reply:
Not sure if its a pro or con but having a single legacy repo seems to take less management overhead for something which we intend to delete ASAP. I worry that having the separated legacy repos may encourage a Laissez-faire attitude.

Reply:
That sounds good to me, will update

Reply:
updated!